compose-builder/add-security.yml

```diff
@@ -229,5 +229,14 @@ services:
       - security-bootstrapper
 
   ui:
-    environment:
-      EDGEX_SECURITY_SECRET_STORE: "true"
```

Comment:
please keep this env, or it will become "false" after compose build
https://github.com/edgexfoundry/edgex-compose/blob/14379f0c784878221e464ab630af5dd041452589/docker-compose.yml#L1376

Reply:
It turns out that also needs to add following config for ui in docker-compose-base.yml to make it work in both secure and no-secure mode.

```
env_file:
      - common-non-security.env
```